ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -164,11 +163,14 @@ open class PeoplePicker: BadgeField {
             self.pickPersona(persona: persona)
         }
         personaListView.searchDirectoryDelegate = self
-        personaListView.backgroundColor = UIColor(light: fluentTheme.color(.background2).light,
-                                                  dark: fluentTheme.color(.background2).dark)
 
         NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardWillHide(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
+
+        // Update appearance whenever `peoplePickerTokenSet` changes.
+        peoplePickerTokenSet.registerOnUpdate(for: self) { [weak self] in
```

Comment:
just trying to understand how the token sets work, so will this be called on init for the initial setting of background color, correct?

Reply:
Sort of. This will get called anytime the token set gets an update, which is usually an override is set, or the theme changes. Usually the theme will change when you move to a new window, so moving to a window will usually trigger this, but there are some cases where the control moves to a window using the default theme, and the update doesn't get called. Our demo app is bad for testing that, since the "default" theme isn't actually the shared default theme. So we should probably be making sure we set the background color in init, also.